remo/featuredrep/views.py

```diff
@@ -11,11 +11,17 @@
 import forms
 from models import FeaturedRep
 
+from datetime import datetime
+
 
 @cache_control(private=True, max_age=60 * 10)
 def list_featured(request):
     """List all Featured Reps."""
-    featured_list = FeaturedRep.objects.all()
+
+    if not request.user.has_perms('featuredrep.can_edit_featured'):
```

Comment:
this always is true as has_perms (at least for me here) always returns false. This is for a user in the "Council" group. I can't tell you right now why though.. But this leads to the following problem:

![screen shot 2018-04-14 at 17 46 14](https://user-images.githubusercontent.com/330324/38769960-c3247720-400b-11e8-96fa-1a0ec261a258.png)

As you can see, I see the button on the top right, but I don't see all existing entries:

![screen shot 2018-04-14 at 17 13 37](https://user-images.githubusercontent.com/330324/38769953-97e615f0-400b-11e8-8eb1-b8c12113382d.png)

In any case, why should people who do not have permission to edit the featured reps only see the ones in the *future*? Shouldn't that be only the ones from today and in the past?

Reply:
The idea was to not show in the list to the people not council the scheduled featured rep.

Reply:
correct, but this means that normal Reps should see everything from today to the past. As it is now, they will see all in the future (and just those). This is 100% the opposite of what we want ;)

Reply:
ops I have to revert but seems easy to fix

Reply:
Can you check the latest commit for this point?

Reply:
As I can see the non-council cannot see the future reps and as council I can see them.
I have an account that is only rep and I see only one.

Reply:
I will test this again.